extensions/spotify-player/src/components/AddToPlaylistAction.tsx

```diff
@@ -15,7 +15,7 @@ export function AddToPlaylistAction({ playlists, meData, uri }: AddToPlaylistAct
   return (
     <ActionPanel.Submenu icon={Icon.List} title="Add to Playlist">
       {playlists
-        ?.filter((playlist) => playlist.owner?.id === meData?.id || playlist.collaborative)
```

Comment:
Why did we remove this?

Reply:
> Yeah makes sense. I believe it's a problem with the collaborative flag. Spotify doesn't update correctly. I would remove now the `|| playlist.collaborative` from both my code and the one in nowPlaying. Would this work? Or I can leave it in and update the error message.
Because of this.

Reply:
Ah ok, got it. thanks! Can we update the changelog to make a note that we removed this? Because I presume that for most users, they will see less playlists now